dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentsExportController.java

```diff
@@ -82,8 +86,24 @@
 
     private final EnrollmentFieldsParamMapper fieldsMapper;
 
+    @Value
+    @OpenApi.Property
+    @OpenApi.Shared( value = false )
+    private static class ObjectListResponse
```

Comment:
I guess the point of such a generic response class which uses the `EntityType` mechanism for "generics" is to be reused in more than one controller. So if this is needed and useful this should be declared outside of the enrolment specific controller. Otherwise this could simply use `List<Enrollment>` instead to be less complex and do the same.

Reply:
You are right and I am going to reuse it. I just did not want to think about were to put it just yet 😂 I can go back to the `List<Enrollment>` and make it generic when I reuse it 👌🏻

Reply:
Its fine to keep it as is until you reuse it, just wanted to make sure you didn't misunderstand the concept.